vmware_dvs/agent/firewalls/dvs_securitygroup_rpc.py

```diff
@@ -51,6 +52,7 @@ def init_firewall(self, defer_refresh_firewall=False):
         # Flag raised when a global refresh is needed
         self.global_refresh_firewall = False
         self._use_enhanced_rpc = None
+        self._dev_to_update = set()
```

Comment:
let's use full version of naming variable:  `self._dev_to_update  -> self._devices_to_update` to increase readability